java/src/com/ibm/streamsx/rest/StreamingAnalyticsConnection.java

```diff
@@ -104,10 +114,11 @@ private void setJobsPath(String sJobs) {
 
     /**
      * Streaming Analytics only allows one instance per service, so each
-     * connection can only ever access a single instance that we've known about
-     * since object creation
+     * connection can only ever access a single instance that we have known
```

Comment:
Who is "we"? As a one sentence summary this is very wordy. Can it be simplified to describe what the method does.